Makefile

```diff
@@ -135,7 +135,7 @@ shell:
 	bash
 
 configs-integration-test:
-	/bin/bash -c "go test -v -tags 'netgo integration' -timeout 30s ./pkg/configs/... ./pkg/ruler/..."
+	/bin/bash -c "go test -v -tags 'netgo requires_docker' -timeout 30s ./pkg/configs/... ./pkg/ruler/..."
```

Comment:
This is a different set of integration tests. You should actually rollback this and replace the `-tags=integration` in `.circleci/config.yml`.

Reply:
ups